<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>459</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Battery Control</string>
  </property>
  <property name="windowIcon">
   <iconset theme="battery">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabs">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Battery Information</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout" rowstretch="40,0">
        <item row="0" column="0">
         <widget class="QWidget" name="remaining_battery" native="true">
          <property name="styleSheet">
           <string notr="true">#remaining_battery {
	border: 1px solid #d2d2d2;
}</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="leftMargin">
            <number>13</number>
           </property>
           <item>
            <widget class="QLabel" name="remaining_label">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Total remaining battery:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="BatteryIcon" name="battery" native="true">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>50</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">#widget {
	margin: 10px;
}</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>123</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QWidget" name="details" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">#details {
	border: 1px solid #d2d2d2;
}</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QLabel" name="battery_details">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Battery details:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <layout class="QHBoxLayout" name="batterychooser">
             <item>
              <widget class="QLabel" name="battery_label">
               <property name="text">
                <string>Battery:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="battery_combo"/>
             </item>
            </layout>
           </item>
           <item row="2" column="0">
            <layout class="QHBoxLayout" name="info_container" stretch="60,40">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="battery_info_label">
               <property name="text">
                <string>Status:
Remaining Percentage:
Remaining capacity:
Full charge capacity:
Current:
Voltage:
Wattage:
Cycle count:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="battery_info">
               <property name="text">
                <string/>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="2" column="1">
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <layout class="QHBoxLayout" name="manu_container" stretch="60,40">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="sizeConstraint">
                <enum>QLayout::SetMinimumSize</enum>
               </property>
               <item>
                <widget class="QLabel" name="battery_manu_label">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>Manufacturer name:
Serial number:
FRU Part number:
Device chemistry:
Design capacity:
Design voltage:</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="battery_manu">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QLabel" name="manu_logo">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>30</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QWidget" name="batteries" native="true">
          <property name="styleSheet">
           <string notr="true">#batteries {
	border: 1px solid #d2d2d2;
}</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="4" column="0" colspan="2">
            <widget class="QPlainTextEdit" name="status">
             <property name="lineWrapMode">
              <enum>QPlainTextEdit::WidgetWidth</enum>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
             <property name="plainText">
              <string/>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="batteries_installed">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Batteries Installed:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="condition">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Condition:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QPushButton" name="maintain">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Battery maintenance</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QWidget" name="primary_battery" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="primary_label">
                <property name="text">
                 <string>1 - Main Battery</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="primary_cont">
                <property name="spacing">
                 <number>6</number>
                </property>
                <property name="leftMargin">
                 <number>5</number>
                </property>
                <item>
                 <widget class="QLabel" name="primary_icon">
                  <property name="enabled">
                   <bool>true</bool>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>20</width>
                    <height>11</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>20</width>
                    <height>11</height>
                   </size>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="pixmap">
                   <pixmap>../../../Desktop/good.png</pixmap>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="primary_text">
                  <property name="text">
                   <string>Good</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item row="2" column="0" colspan="2">
            <widget class="QWidget" name="secondary_battery" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="secondary_label">
                <property name="text">
                 <string>2 - Seconday Battery</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="secondary_cont">
                <property name="spacing">
                 <number>6</number>
                </property>
                <property name="leftMargin">
                 <number>5</number>
                </property>
                <item>
                 <widget class="QLabel" name="secondary_icon">
                  <property name="minimumSize">
                   <size>
                    <width>20</width>
                    <height>11</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>20</width>
                    <height>11</height>
                   </size>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="pixmap">
                   <pixmap>../../../Desktop/bad.png</pixmap>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="secondary_text">
                  <property name="text">
                   <string>Poor</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="bottom_container">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="thinkpads_logo">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../resources.qrc">:/res/logo.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="close_button">
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>He&amp;lp</string>
    </property>
    <addaction name="actionVisit_Thinkpads_org"/>
    <addaction name="separator"/>
    <addaction name="actionAbout_Battery_Control"/>
   </widget>
   <addaction name="menuAbout"/>
  </widget>
  <action name="actionVisit_Thinkpads_org">
   <property name="text">
    <string>&amp;Visit Thinkpads.org</string>
   </property>
  </action>
  <action name="actionAbout_Battery_Control">
   <property name="text">
    <string>&amp;About Battery Control</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BatteryIcon</class>
   <extends>QWidget</extends>
   <header>ui/batteryicon.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>battery_combo</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>displaySelectedBattery(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>283</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>close_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>750</x>
     <y>431</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAbout_Battery_Control</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>openAbout()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionVisit_Thinkpads_org</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>openSite()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>displaySelectedBattery(QString)</slot>
  <slot>openAbout()</slot>
  <slot>openSite()</slot>
 </slots>
</ui>
